docs/Manifest.toml

```diff
@@ -1,24 +1,20 @@
+# This file is machine-generated - editing it directly is not advised
+
 [[Base64]]
 uuid = "2a0f44e3-6c83-55bd-87e4-b1978d98bd5f"
 
-[[ChainRules]]
```

Comment:
This fixes the failure see here https://travis-ci.org/JuliaDiff/ChainRulesCore.jl/jobs/655949602

But it's not sufficient to make documentation build, since doctests also needed fixing